### PR TITLE
Landing page: Adding stay-productive-section

### PR DESCRIPTION
**Pull request by jrukundo256**
### Description

_Landing page: Add **Stay Productive** section_

### Changelog

_Brief summary on the technical changes in this PR_
 - _Adding HTML and appropriate styling_
 - _Yet to make it responsive_

### How to test
Open locally in your preferred browser
image

### CODE REVIEW

**Comment by Famba-Abdulnassiir** [comment]
Thanks for this its nice work and clean code, lets wait for one more review so we can merge it.

File: style.css
Comment (MosesMusoke) [inline style.css:1]: nice work

File: index.html
Comment (MosesMusoke) [inline index.html:1]: nice work

**Review by MosesMusoke — APPROVED** [review]
nice work

**Review by Famba-Abdulnassiir — APPROVED** [review]

**Pull request merged**
